ct/2fauth.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: jkrgr0
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://docs.2fauth.app/
+
+# App Default Values
+APP="2FAuth"
+TAGS="2fa;authenticator"
+var_cpu="1"
+var_ram="512"
+var_disk="2"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+
+# App Output & Base Settings
+header_info "$APP"
+base_settings
+
+# Core
+variables
+color
+catch_errors
+
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+
+    # Check if installation is present | -f for file, -d for folder
+    if [[ ! -d "/opt/2fauth" ]]; then
+        msg_error "No ${APP} Installation Found!"
+        exit
+    fi
+
+    # Crawling the new version and checking whether an update is required
+    RELEASE=$(curl -s https://api.github.com/repos/Bubka/2FAuth/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+    if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+        msg_info "Updating $APP"
```

Comment:
```suggestion
       
```

Reply:
Shouldn't we keep it, as this indicates that the update is processing?

Reply:
You have it down below at line 52 anyway. And very message_info needs a messge_ok, or else de spinner will not be deleted an will spawn a second time.

Reply:
Correct, but on line 52 the "inner" update of the app itself is performed with it's own `msg_ok` in line 59 (added in my latest changes, since it was missing before).  

The `msg_info "Updating $APP"` on line 41 is the "outer" update message for the whole update process, which is being cleared on line 79 with `msg_ok "Update Successful"`.

Reply:
Oh,did not see that a first sight. Bit its a bit redundent. Could ypu make the first one like the second and lose that instead?